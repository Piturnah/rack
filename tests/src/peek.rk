fn main in
  1 2 peek a b in a print b print a print end 
  "---------------\n" puts
  print print
end
