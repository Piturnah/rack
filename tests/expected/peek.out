----STDOUT----
1
2
1
---------------
2
1

----STDERR----
